src/include/update/rootpart.rb

```diff
@@ -146,12 +146,17 @@ def make_partition_list(withall, flavor)
           fs = ""
 
           # is a linux fs, can be a root fs, has a fs name
-          if part_fs != nil && Ops.get(i, :fstype) != nil &&
-              (CanBeLinuxRootFS(part_fs) || legacy_filesystem?(part_fs)) &&
-              part_fs_name != nil
+          if part_fs != nil &&
+            (CanBeLinuxRootFS(part_fs) || legacy_filesystem?(part_fs)) &&
+            part_fs_name != nil
+            # We are sure that we have found a valid linux partition for update.
+            # In some cases fstype has not been set correctly while previous
+            # installation. E.g. Parted (versions older than 3.2) has set
+            # fstype to "Microsoft Basic Data" although it is a linux partition.
+            # So we are not showing this entry in order to not confusing the
+            # user.
             fs = Builtins.sformat(
-              _("%1 (%2)"),
-              Ops.get_string(i, :fstype, ""),
+              _("Linux (%1)"),
```

Comment:
I think displaying fixed `Linux` string is pointless, but to not break the behavior for SLE15 I'm fine with it. For `master` I'd probably just display the fs name, e.g. `fs = part_fs_name`.